index.html

```diff
@@ -450,13 +450,13 @@ <h3>Abstract Roles</h3>
 		</section>
 		<section id="requiredState">
 			<h3>Required States and Properties</h3>
-			<p><a>States</a> and <a>properties</a> specifically required for the <a>role</a> and subclass roles. Content authors MUST provide a non-empty <span>value</span> for required states and properties. Content authors MUST NOT use the value <code>undefined</code> for required states and properties, unless <code>undefined</code> is an explicitly-supported value of that state or property.</p>
+			<p><a>States</a> and <a>properties</a> specifically required for the <a>role</a> and subclass roles. Content authors MUST provide a non-empty <span>value</span> for required states and properties, except if current value is the default for that state or property for the role. Content authors MUST NOT use the value <code>undefined</code> for required states and properties, unless <code>undefined</code> is an explicitly-supported value of that state or property.</p>
 			<p>When an <a>object</a> inherits from multiple ancestors and one ancestor indicates that property is supported while another ancestor indicates that it is required, the property is required in the inheriting object.</p>
 			<p class="note">A host language attribute with the appropriate <a href="#implicit_semantics">implicit WAI-ARIA semantic</a> fulfills this requirement.</p>
 		</section>
 		<section id="supportedState">
 			<h3>Supported States and Properties</h3>
-			<p><a>States</a> and <a>properties</a> specifically applicable to the <a>role</a> and child roles. <a>User agents</a> MUST map all supported states and properties for the role to an accessibility API. Content authors MAY provide <span>values</span> for supported states and properties, but need not in some cases where default values are sufficient.</p>
+			<p><a>States</a> and <a>properties</a> specifically applicable to the <a>role</a> and child roles. <a>User agents</a> MUST map all supported states and properties for the role to an accessibility API. If the state or property is undefined, and it has a default value for the role, <a>user agents</a> MUST use the default value. Content authors MAY provide <span>values</span> for supported states and properties, but need not in some cases where default values are sufficient.</p>
```

Comment:
Editorial nit: "use the default value" perhaps should be "expose the default value" or something that clarifies what "use" means.

Non-editorial issue: This introduces a normative change and because it's a MUST, we will need tests and at least two passing implementations -- possibly for all cases where a state or property has a default value....

Possible solution: Changing "MUST" to "SHOULD" means we're accomplishing the "clarify" part by stating what our expectations are. If we truly want it to be a MUST, then perhaps do that in 1.3?

Reply:
This reorders the sentences to (hopefully) address @jnurthen's comment in https://github.com/w3c/aria/pull/1163#discussion_r369301194 so that:
1. _first_, authors are told that they may set a value,
2. _then_ user agents are told that (in addition to mapping the attr to an api), if the authors don't set a value (i.e. it's undefined), they should expose the default value (if there is one for the role)

Also, addresses @joanmarie's concern about MUST by using SHOULD. (We can discuss whether or not to make this a normative requirement in 1.3.) Also, this changes "use the default value" to "expose the default value" per joanie's request.

I also deleted "some" from "in some cases" because I like deleting words. ;)

```suggestion
			<p><a>States</a> and <a>properties</a> specifically applicable to the <a>role</a> and child roles. Content authors MAY provide <span>values</span> for supported states and properties, but need not in cases where default values are sufficient. <a>User agents</a> MUST map all supported states and properties for the role to an accessibility API. If the state or property is undefined and it has a default value for the role, <a>user agents</a> SHOULD expose the default value.</p>
```

Reply:
I think the "User agents MUST map all supported states and properties" is not necessary and might not be desirable:

* User agents already need to expose supported states and properties as stated in the Core-AAM. Thus saying in the ARIA spec that they need to do so doesn't seem to buy us anything.
* There are instances in the Core-AAM where the mapping is "not mapped" because the correct "mapping" is the absence of exposing anything. For instance if `aria-haspopup` is set to `false`, you don't map on most platforms because platforms don't have a property or state which means "this thing doesn't have a popup." So In this particular case, what are we telling user agents to do when we say they "MUST map all supported states and properties?"
* If you add a "user agents MUST", I believe we need to add tests. And this particular "user agents MUST" seems pretty broad. We have a lot of roles and a lot of states and properties. And now we're talking about the combination of "all supported states and properties" for (I assume) all roles which have supported states and properties.

That said, you would not have included it if you didn't think it accomplished something. So please explain.